### DISCUSSION TITLE
Can I get WT to use a specific font for a specific application?

### DISCUSSION THREAD

**Discussion by MrDowntempo** [wezterm/wezterm#6167] [Q&A]
### What Operating System(s) are you running on?

Linux X11, Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Currently using plasma on KDE5 (not the lastest) because I'm on debian stable.

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Wezterm's font rendering is nifty. Other terminal emulators don't seem to support multiple fonts at the same time, But in Wezterm I've got it using a different font for italics for instance, and it works perfectly. It gives my comments a handwritten look in editors.

I was curious to see if I could also get wezterm to use certain fonts for certain applications. I didn't notice any support for that in the docs though. Then I could use an old DOS terminal font for roguelikes and DOS games, a more readable font for apps with paragraphs of text, a matrix font for cmatix, and silly stuff like that etc.

**Comment by loops** [comment]
Hey MrDowntempo,

You're probably looking for something fancier than this, but you could just have an alternate config file, set up with the alternate fonts you want.  When you start WT for your DOS game, just specify the alternate config file on the command line.

It's not integrated into a single running instance, but it'd get you most of the way to what you want.
